library/ui-styles/src/main/res/values/theme_dark.xml

```diff
@@ -129,7 +129,9 @@
         <item name="android:windowSharedElementEnterTransition">@transition/image_preview_transition</item>
         <item name="android:windowSharedElementExitTransition">@transition/image_preview_transition</item>
 
-        <item name="vctr_social_login_button_google_style">@style/Widget.Vector.Button.Outlined.SocialLogin.Google.Dark</item>
+        <!-- For Google button style, use same values than for light theme, for a better rendering (white background)
+        see https://github.com/vector-im/element-android/issues/4285#issuecomment-974270998 -->
+        <item name="vctr_social_login_button_google_style">@style/Widget.Vector.Button.Outlined.SocialLogin.Google.Light</item>
```

Comment:
@ouchadam not sure if it is better to use light style here, or to update the Dark style itself at https://github.com/vector-im/element-android/blob/main/library/ui-styles/src/main/res/values/styles_social_login.xml#L32, to just basically copy paste values from the light style.

We could also have only one single style for Google button, but I'd like to keep the same arch for all sso button styles.

It was my first idea, but I changed my mind. What would you do?

Reply:
I prefer this way as we're explicitly overriding the dark behaviour for the social google style in the context of the page, rather than the button itself